src/components/ContributorsWall.tsx

```diff
@@ -0,0 +1,19 @@
+import { type Library } from '~/libraries'
+
+export function ContributorsWall({ library }: { library: Library }) {
+  return (
+    <div className="flex flex-col items-center my-4">
+      <a
+        href={`https://github.com/tanstack/${library.id}/graphs/contributors`}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <img
+          alt="GitHub Contributors"
+          src={`https://contrib.rocks/image?repo=tanstack/${library.id}`}
```

Comment:
Since we are using Contrib Rocks, perhaps we can credit them with a hyperlink paired with some simple verbiage.

- "generated using contrib.rocks"
- "made by contrib.rocks"